### PR TITLE
Implemented getting flags by identity and setting traits at the same time

### PR DESCRIPTION
**Pull request by jackforesightmobile**

### CODE REVIEW

File: FlagsmithClient/Classes/Flagsmith.swift [hunk 1]
```diff
@@ -268,6 +268,21 @@ public final class Flagsmith: @unchecked Sendable {
...
+    
+    /// Get flags for an identity and set traits in the same call
+    ///
+    /// - Parameters:
+    ///   - traits: Traits to be created or updated
+    ///   - identity: ID of the user
+    ///   - completion: Closure with Result which contains a list of Flags in case of success or Error in case of failure
+    public func getIdentityFlags(_ traits: [Trait],
+                          forIdentity identity: String,
+                          completion: @Sendable @escaping (Result<[Flag], any Error>) -> Void)
```
Comment (gazreese) [inline FlagsmithClient/Classes/Flagsmith.swift]: You'll have to fix this swift lint things. If you run it locally there's --fix option that sorts most of it

File: FlagsmithClient/Classes/Flagsmith.swift [hunk 2]
```diff
@@ -268,6 +268,21 @@ public final class Flagsmith: @unchecked Sendable {
...
+    {
+        apiManager.request(.postTraits(identity: identity, traits: traits)) { (result: Result<Traits, Error>) in
```
Comment (gazreese) [inline FlagsmithClient/Classes/Flagsmith.swift:282]: Need to do something with the error?

Comment (jackforesightmobile) [reply FlagsmithClient/Classes/Flagsmith.swift:282]: I've just mirrored the other methods that do things with traits

File: FlagsmithClient/Classes/Flagsmith.swift (hunk 1 above)
Comment (jackforesightmobile) [reply FlagsmithClient/Classes/Flagsmith.swift]: Thanks will sort

**Pull request merged**